proto/pdpb.proto

```diff
@@ -433,6 +433,11 @@ message SplitRegion {
     repeated bytes keys = 2;
 }
 
+message SwitchWitness {
+    repeated uint64 peer_id = 1;
```

Comment:
Seems extract them as a struct and make the struct repeated is a better style. But this is also OK.

Reply:
Ok, I plan to change it to `SwitchWitness` and `BatchSwitchWitness` like `Split` and `BatchSplit`.